spec/features/pages/escort.rb

```diff
@@ -60,9 +60,9 @@ def confirm_read_only_move_details
       end
     end
 
-    def confirm_detainee_details(detainee)
+    def confirm_detainee_details(detainee, police: false)
```

Comment:
The escort class now has a method called number:
```
  def number
    return pnc_number if from_police?
    prison_number
  end
```

maybe we could check just for that one #